baseband/gsb/tests/test_gsb.py

```diff
@@ -470,25 +480,27 @@ def test_raw_stream(self, tmpdir):
             # For testing proper file closing below.
             header0 = fh_r.header0
 
-        with gsb.open(SAMPLE_RAWDUMP_HEADER, mode='rs', raw=SAMPLE_RAWDUMP,
+        with gsb.open(SAMPLE_RAWDUMP_HEADER, 'rs', raw=SAMPLE_RAWDUMP,
+                      sample_rate=sample_rate,
                       samples_per_frame=2**13) as fh_r, \
                 open(str(tmpdir.join('test_time.timestamp')), 'w+b') as sh,\
                 open(str(tmpdir.join('test.dat')), 'w+b') as sp:
             fh_w = gsb.open(sh, 'ws', raw=sp, samples_per_frame=2**13,
```

Comment:
I'd do `self.payloadsize * (8 // bps)` (here and below) but it is not a big deal.